shared/fs/common/index.stories.js

```diff
@@ -19,6 +19,7 @@ import {type OwnProps as PathItemIconOwnProps} from './path-item-icon-container'
 import {type OwnProps as PathItemInfoOwnProps} from './path-item-info-container'
 import SyncStatus from './sync-status'
 import PieSlice from './pie-slice'
+import ReallyDelete from '../really-delete'
```

Comment:
Nit: Would you mind either moving `fs/really-delete` to `fs/common/really-delete`, or make a separate stories file in `fs/really-delete/index.stories.js`? It'd be nice to keep this story file only for fs/common components.